File: February 22, 2026.js
function medalTable(results) {
  const medals = {};

  for (const [gold, silver, bronze] of results) {
    if (!medals[gold]) medals[gold] = { gold: 0, silver: 0, bronze: 0, total: 0 };
    if (!medals[silver]) medals[silver] = { gold: 0, silver: 0, bronze: 0, total: 0 };
    if (!medals[bronze]) medals[bronze] = { gold: 0, silver: 0, bronze: 0, total: 0 };

    medals[gold].gold++;
    medals[gold].total++;

    medals[silver].silver++;
    medals[silver].total++;

    medals[bronze].bronze++;
    medals[bronze].total++;
  }

  const sortedCountries = Object.keys(medals).sort((a, b) => {
    if (medals[b].gold !== medals[a].gold) {
      return medals[b].gold - medals[a].gold;
    }
    return a.localeCompare(b);
  });

  
  let csv = "Country,Gold,Silver,Bronze,Total";

  for (const country of sortedCountries) {
    const m = medals[country];
    csv += `\n${country},${m.gold},${m.silver},${m.bronze},${m.total}`;
  }

  return csv;
}

File: February 22, 2026.py
def count_medals(winners):

    medals = {}
    countries = []

    for gold, silver, bronze in winners:
        if gold not in medals:
            medals[gold] = [0, 0, 0]
        if silver not in medals:
            medals[silver] = [0, 0, 0]
        if bronze not in medals:
            medals[bronze] = [0, 0, 0]

        medals[gold][0] += 1
        medals[silver][1] += 1
        medals[bronze][2] += 1

    
    for country, (gold, silver, bronze) in medals.items():
        total = gold + silver + bronze
        countries.append((country, gold, silver, bronze, total))

    countries.sort(key=lambda x: (-x[1], x[0]))

    lines = ["Country,Gold,Silver,Bronze,Total"]
    for country, gold, silver, bronze, total in countries:
        lines.append(f"{country},{gold},{silver},{bronze},{total}")

    return "\n".join(lines)